### PR TITLE
(Wix) Move pdb files to their own Feature

### PR DESCRIPTION
**Pull request by sblaisot**
This will move the PDB files, when bundle_pdb=1, to their own Feature so you can choose at Mixxx installation if you want to install PDB files or not. PDB installation is enabled by default.

**/!\ WARNING /!\**
This PR should be merged after #862

### CODE REVIEW

**Comment by rryan** [comment]
I don't know much about how Wix works yet -- does the option get hidden when Mixxx was built with bundle_pdbs=0?

**Comment by sblaisot** [comment]
ok, let me explain a bit more:

* Before This PR
- when bundle_pdbs=0, nothing happens (pdbs are not bundled in the installer, are not installed on the target system)
- when bundle_pdbs=1, pdb files are bundled in the installer and silently installed with Mixxx without any advertisement or choice to not install them

* After this PR
- when bundle_pdbs=0, there is no change, pdbs are not included
- when bundle_pdbs=1, a new Feature is shown in the installer, activated by default, about installing pdb files. The user installing Mixxx can choose to install PDB files or not. see screenshot below.

mixxx setup

File: build/wix/Localization/mixxx_ca-ES.wxl [hunk 1]
```diff
@@ -39,6 +39,8 @@
   <String Id="FeatureCompleteDescription">El paquet complet.</String>
   <String Id="FeatureMainProgramTitle">L'aplicació.</String>
   <String Id="FeatureMainProgramDescription">L'aplicació Mixxx incloent les configuracions per defecte de diversos controladors.</String>
+  <String Id="FeaturePDBTitle">PDB debug files</String>
+  <String Id="FeaturePDBDescription">Install Mixxx PDB files to use with a debugger. This is usefull to debug Mixxx and provide readable backtrace to the development team.</String>
```
Comment (rryan) [inline build/wix/Localization/mixxx_ca-ES.wxl]: nit:
usefull -> useful
backtrace -> backtraces

Comment (sblaisot) [inline build/wix/Localization/mixxx_ca-ES.wxl]: ok, updated. thx.

**Comment by rryan** [comment]
LGTM thank you :) I guess we'll wait on #862. Hopefully @Pegasus-RPG can give it a try on the new server -- if it works I'd say just merge it and we'll work out any kinks that arise after.

**Comment by sblaisot** [comment]
Closing this PR which commits have been cherry-picked into #862

**Pull request closed without merging**